app/renderer/components/navigation/navigator.js

```diff
@@ -257,9 +263,24 @@ const styles = StyleSheet.create({
     left: 'auto',
     right: '2px'
   },
+
   braveMenuContainer: {
     position: 'relative'
   },
+
+  braveMenuContainer__braveShield_down: {
+    filter: 'grayscale(100%)'
+  },
+
+  braveMenuContainer__braveShield_disabled: {
+    filter: 'grayscale(100%)',
```

Comment:
grayscale ftw

Reply:
@luixxiul does this also fix https://github.com/brave/browser-laptop/issues/9696?

Reply:
I don't think so as this does not change the color of the number block.

Reply:
I have a fix for that issue. Is it fine to commit it to this PR?

<img width="34" alt="screenshot 2017-09-22 16 20 15" src="https://user-images.githubusercontent.com/3362943/30733464-ffc7ba16-9fb1-11e7-8452-27e6a64224ef.png">

Reply:
Sure- add it in 😄

Reply:
Added :-) https://github.com/brave/browser-laptop/pull/10951/commits/8615b5fb832965d72cb4072ecc420dc47fb04d9e